### PR TITLE
terms and conditions

### PR DESCRIPTION
**Pull request by garr741**

### CODE REVIEW

File: imports/startup/client/routes.js [hunk 1]
```diff
@@ -35,6 +36,7 @@ export const Routes = () => (
         <Route path=":id" component={UserPage}/>
       </Route>
       <Route path="testsso" component={SsoButtons} />
+      <Route path="termsandconditions" component={TermsAndConditions} />
```
Comment (ashleyzeg) [inline imports/startup/client/routes.js:39]: camel case?

File: imports/ui/components/Footer.jsx [hunk 2]
```diff
@@ -9,6 +10,7 @@ class Footer extends Component {
           <span>&copy; 2017 All Rights Reserved</span>
           <span>United Way of Central Indiana</span>
           <span>Website by Indy Tech Fellowship</span>
+          <Link to={'/termsandconditions'}>Terms and Conditions</Link>
```
Comment (ashleyzeg) [inline imports/ui/components/Footer.jsx:13]: were you going to use the browseHistory.push() here onClick instead of the Link?

**Review by bwencke — APPROVED** [review]

**Review by ashleyzeg — COMMENTED** [review]
few questions/comments but otherwise looks good

File: imports/startup/client/routes.js (hunk 1 above)
Comment (garr741) [reply imports/startup/client/routes.js:39]: noThanks

**Review by ashleyzeg — APPROVED** [review]

**Pull request merged**